README.md

```diff
@@ -220,21 +220,34 @@ This plugin creates device plugin endpoints based on the configurations given in
+|------------------|----------|----------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------|-----------------------------------------------------------------|
 | "resourceName"   | Y        | Endpoint resource name. Should not contain special characters including hyphens and must be unique in the scope of the resource prefix | string                                                | "sriov_net_A"                                                   |
-| "resourcePrefix" | N        | Endpoint resource prefix name override. Should not contain special characters                                     | string Default : "intel.com"                          | "yourcompany.com"                                               |
-| "deviceType"     | N        | Device Type for a resource pool.                                                                                  | string value of supported types. Default: "netDevice" | Currently supported values: "accelerator", "netDevice"          |
-| "excludeTopology" | N       | Exclude advertising of device's NUMA topology          | bool Default: "false"    | "excludeTopology": true          |
-| "selectors"      | N        | A map of device selectors. The "deviceType" value determines the "selectors" options.                             | json object as string Default: null                   | Example: "selectors": {"vendors": ["8086"],"devices": ["154c"]} |
+| "resourcePrefix" | N        | Endpoint resource prefix name override. Should not contain special characters                                                          | string Default : "intel.com"                          | "yourcompany.com"                                               |
+| "deviceType"     | N        | Device Type for a resource pool.                                                                                                       | string value of supported types. Default: "netDevice" | Currently supported values: "accelerator", "netDevice"          |
+| "excludeTopology" | N       | Exclude advertising of device's NUMA topology                                                                                          | bool Default: "false"    | "excludeTopology": true          |
+| "selectors"      | N        | A map of device selectors. The "deviceType" value determines the "selectors" options.                                                  | json object as string Default: null                   | Example: "selectors": {"vendors": ["8086"],"devices": ["154c"]} |
+| "additionalEnvVariables" | N | A map of map to add additional environment variables to devices                                                                        | json object as string Default: null  | Example: "additionalEnvVariables": {"*": {"token": "3e49019f-412f-4f02-824e-4cd195944205"}} |
```

Comment:
Can you add a dedicated section about this one ?

explaining what it does, how to use it with some examples ?

also explaining the expected keys e.g that `"*"` means for all devices and device ID (e,g `"0000:03:00.5"`) for the specific device. plus that the latter overwrites values of the former.